README.md

```diff
@@ -96,16 +92,24 @@ To install the Foundry development toolchain to have `cast` available, check the
 
 ### Gathering DApp data
 
-The DApp contract address is required for all interactions with the Honeypot.
-In order to gather the address used in a local deployment, first [start up the DApp](./#running-the-application).
-Then, execute the following command from the current directory to extract it from the DApp deployment data:
+Some contract addresses are required for interactions with the Honeypot.
+
+In order to gather them, first [start up the DApp](./#running-the-application).
+Then, execute the following command from the current directory to extract the data from the DApp deployment data for a given `$NETWORK`:
 
 ```shell
-export DAPP_ADDRESS=$(cat deployments/localhost/dapp.json | \
-                        jq -r '.address')
+export DAPP_ADDRESS=$(cat deployments/$NETWORK/dapp.json | jq -r '.address')
```

Comment:
For testnets, looks like you need to use `honeypot.json`. This is a legacy of the examples (which needed to have different names for the dapps). I guess you could either always use `honeypot.json` or `dapp.json`.

Reply:
Yes, I failed to adjust that before. 

Fixed at 1f995ac.

Reply:
In that commit it doesn't make sense to simply change the README: if you want to use `honeypot.json` for both local and testnet (I like the idea btw), you need to change some things around in docker-compose.yml.

Reply:
@miltonjonat please check wether the following commits address your concerns:
- a900c03
- a6b5fbc (the one I referred to above, https://github.com/cartesi/honeypot/commit/1f995acaa1b9b699285839553097a0582c3d2550, has the same changes but is oudated)
- 8ffb88f (new commit, with some reworkding)

Reply:
Sorry about the confusion. Change reverted at a0c1367f779c0, @miltonjonat